recipes-multimedia/picamera-libs/picamera-libs.bb

```diff
@@ -9,6 +9,7 @@ include recipes-bsp/common/raspberrypi-firmware.inc
 S = "${WORKDIR}/raspberrypi-firmware-1.${PV}"
 
 do_install(){
+    rm -f ${S}/opt/vc/lib/libGLES* ${S}/opt/vc/lib/libEGL* ${S}/opt/vc/lib/libWFC.so ${S}/opt/vc/lib/libOpenVG.so
```

Comment:
Shouldn't this `rm` come after the `install` command bellow?

Reply:
I'm not experienced with bitbake, but I thought this was pretty much the same thing. My understanding is that the firmware files are extracted in ${S}/opt/vc/lib, and then all copied to the sysroot.  I thought we could copy and remove or simply remove and not copy at all.

Reply:
The install rule is the one you have in the bb file. There is nothing before (as this is not an append). So, I'm not sure what the aim is here. Have you tried your approach?

Reply:
It seemed to solve the problem for me. The wrong libGLES library was not in the sysroot anymore.
Did you have a different result?

Reply:
Right. I missed one thing - that this removes from S. I would avoid that mutating S. Maybe clean up the install later or do an rm on D after things get installed.

Reply:
I changed as you requested. It seems to still work for me.